jenkins/continuous-upgrade.groovy

```diff
@@ -32,16 +34,26 @@ pipeline {
             when {
                 branch 'master'
             }
-            steps {
```

Comment:
You shouldn't need to change this at all. The post section should just be nested within `pipeline { }` with just the notifiy stuff in it, none of the stuff you moved around.